src/types/systemIntake.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Type for SystemIntakeForm
+ *
+ */
+export type SystemIntakeForm = {
```

Comment:
My initial idea on a model for the System Intake. We can continue to refine names if these don't work.